doc/openapi.yaml

```diff
@@ -73,10 +71,77 @@ paths:
                     $ref: '#/components/schemas/Container'
         401:
           $ref: '#/components/responses/UnauthorizedError'
-  /paymodels:
+  /options?id=foobar:
```

Comment:
i ended up not using this new endpoint for this feature, but since it was already working and tested, i left it in

Reply:
This is cool 👍